server.go

```diff
@@ -1039,11 +1039,18 @@ func newServer(cfg *Config, listenAddrs []net.Addr, chanDB *channeldb.DB,
 				return defaultConf
 			}
 
+			minConf := uint64(3)
+			maxConf := uint64(6)
+
+			// If this is a wumbo channel, then we'll require the
+			// max amount of confirmations.
+			if chanAmt > MaxFundingAmount {
```

Comment:
are users able to override these defaults manually?

Reply:
As mentioned offline, they're able to use `--defaultchanconfs` and `--defaultremotedelay` when _starting_ up `lnd` to modify these values. However that'll apply for _all_ channels created from there onwards, as they can't currently specify the values when _creating_ the new channel.